libs/fabric/src/lib/components/combo-box/directives/combo-box-option.directive.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Directive, Input } from '@angular/core';
+import { IComboBoxOption } from 'office-ui-fabric-react';
+
+/**
+ * Wrapper directive for creating a ComboBoxOption
+ */
+@Directive({ selector: 'fab-combo-box-option' })
+export class ComboBoxOptionDirective {
+  @Input() optionKey: IComboBoxOption['key'];
+  @Input() text: IComboBoxOption['text'];
+  @Input() title?: IComboBoxOption['title'];
+  @Input() itemType: IComboBoxOption['itemType'];
+  @Input() index?: IComboBoxOption['index'];
+  @Input() ariaLabel?: IComboBoxOption['ariaLabel'];
+  @Input() selected?: IComboBoxOption['selected'];
+  @Input() disabled?: IComboBoxOption['disabled'] = false;
```

Comment:
why the need for a default value? Let `office-ui-fabric-react` handle it.

Reply:
Concur.